openconcept/aerodynamics/drag_BWB.py

```diff
@@ -0,0 +1,193 @@
+"""
+@File    :   drag_jet_transport.py
+@Date    :   2023/03/23
+@Author  :   Eytan Adler
+@Description : Zero lift drag coefficient buildup for tube-and-wing jet transport aircraft
+"""
+
+# ==============================================================================
+# Standard Python modules
+# ==============================================================================
+
+
+# ==============================================================================
+# External Python modules
+# ==============================================================================
+import openmdao.api as om
+
+# ==============================================================================
+# Extension modules
+# ==============================================================================
+from openconcept.utilities import AddSubtractComp, ElementMultiplyDivideComp
+from openconcept.aerodynamics.drag_jet_transport import SkinFrictionCoefficient_JetTransport, FlapDrag_JetTransport
+
+
+class ParasiteDragCoefficient_BWB(om.Group):
+    """
+    Zero-lift drag coefficient buildup for BWB based on
+    a combination of methods from Roskam, Raymer, and the approach presented by
+    OpenVSP (https://openvsp.org/wiki/doku.php?id=parasitedrag). See the individual
+    component docstrings for a more detailed explanation.
+
+    NOTE: This component does not include drag from the wing, because it assumes
+          it is computed by OpenAeroStruct. Add it in!
```

Comment:
I'm not sure where I would raise that error because it would need to be at the top model level to be able to determine if both groups are included. I'll add a note in the docs though to make sure it's clear you need both.